src/forms/FormFieldColorPicker/ColorPicker.styled.tsx

```diff
@@ -11,7 +11,18 @@ export const ColorContainer = styled.div`
 `;
 
 export const ColorDiv = styled.div`
-  ${pr => `background: rgba(${pr.color.r}, ${pr.color.g}, ${pr.color.b}, ${pr.color.a});`}
+  ${({ color }) => {
+    if (

```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 4 spaces.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9840400)
```suggestion
		if (
```